speaker-segmentation/seg_pyannote.py

```diff
@@ -73,3 +73,66 @@
         RttmSeq(segments).write(os.path.join(args.base_dir, "results",
                                              "pyannote", f"pa_ami_micro_test_sample_{i}.rttm"))
         print(f"Processed sample {i}: {filename}")
+"""
+
+# 13/05/2022 latest edit
+
+# function to filter out segments whose duration is below a threshold
+def duration_selection(segments, threshold):
```

Comment:
For consistency, I would suggest naming all functions as actions (verbs). They should also clearly indicate their purpose (makes the code easier to read).
```suggestion
def remove_short_segments(segments, threshold):
```